Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    rspec-que (1.0.0)
+    rspec-que (1.0.1)
```

Comment:
I take it that this doesn't break what we already have, but only adds new features?